### ISSUE TITLE
 Avoid tabbing inside calendar

### ISSUE THREAD

**Issue by silvia-ribeiro** [wojtekmaj/react-date-picker#308]
Hello, 
Is there a way to avoid tab key navigation inside the calendar? So that when you tab inside the date picker last input (in our case 'year') it ignores all elements inside the calendar and goes to the next tabbable element?

Our case is the following. We have two react date pickers for a "from-to" case, but they need to be independent, it's not a range date picker. When finishing writing the date on the first one and tabbing, it needs to go to the first input (day) of the next date picker, not inside de calendar.

<img width="350" alt="Screenshot 2020-09-21 at 18 40 13" src="https://user-images.githubusercontent.com/51328396/93805091-21e4bf80-fc3f-11ea-8371-1753510e86c0.png">

In summary, we need to block the navigation inside the calendar. Is this possible?

**Comment by BernardoMG** [comment]
+1

**Comment by RicCastelhano** [comment]
+1

**Comment by kabulzhan** [comment]
Has this issue been resolved somehow? The Tab navigation is highly unintuitive.

**Comment by silvia-ribeiro** [comment]
@kabulzhan I don't think there is a solution yet, and we haven't found one either.

**Comment by jdm000** [comment]
This is big.  I wanted to use this component, but because of accessibility concerns (tabbing order being one of them) I had to disable your component today to meet our release date.

**Comment by kabulzhan** [comment]
> 
> 
> This is big. I wanted to use this component, but because of accessibility concerns (tabbing order being one of them) I had to disable your component today to meet our release date.

What other component did you use instead?

**Comment by jdm000** [comment]
> > This is big. I wanted to use this component, but because of accessibility concerns (tabbing order being one of them) I had to disable your component today to meet our release date.
> 
> What other component did you use instead?

Sorry poor wording on my part.  I didn’t disable the whole component but the calendar part of the component.

**Comment by kabulzhan** [comment]
+1

**Comment by netraagal** [comment]
Still nothing ? The tab navigation is quite a problem on the forms...

I'm gonna try to find a way to implement that, will see if luck is with me

**Comment by netraagal** [comment]
Found a solution by using the `onKeyDown` attribute. I'm using some regex to still be able to tab through the year/month/day inputs before going to the next form input.

It will create a strange effect on the back Tab, but for now that's enough for me.

```
import DatePicker from "react-date-picker/dist/entry.nostyle";

<DatePicker
    onKeyDown={ (event:React.BaseSyntheticEvent<KeyboardEvent>) => {
        if (event.nativeEvent.key == 'Tab') {
            const form = event.target.form;
            if (form) {
                let index = [...form].indexOf(event.target) + 1;
                let element, classes;
                for (let i = index; i < form.length; i++ ) {
                    element = form.elements[i];
                    classes = element.classList.toString();
                    if (classes.match(/react-date-picker__inputGroup__(year|month|day)/) ||
                        !classes.match(/react-(calendar|date-picker)/)) {
                        index = i;
                        break;
                    }
                }
                form.elements[index].focus();
                event.preventDefault();
            }
        }
    }}
/>
```
